src/gui/symbology/qgsstyleexportimportdialog.cpp

```diff
@@ -67,7 +67,7 @@ QgsStyleExportImportDialog::QgsStyleExportImportDialog( QgsStyle *style, QWidget
 
   if ( mDialogMode == Import )
   {
-    setWindowTitle( tr( "Import Symbol(s)" ) );
+    setWindowTitle( tr( "Import Symbols or Color Ramps" ) );
```

Comment:
@DelazJ , I'm wondering whether we should rename this to "Import Item(s)" instead, slightly more future-proof (if we ever add saved labelling styles, etc.) Same goes for "Export Symbol(s)".